docs/Targets.md

```diff
@@ -90,6 +96,8 @@ public var sampleData: NSData {
         return "{\"login\": \"\(name)\", \"id\": 100}".dataUsingEncoding(NSUTF8StringEncoding)!
     case .UserRepositories(let name):
         return "[{\"name\": \"Repo Name\"}]".dataUsingEncoding(NSUTF8StringEncoding)!
+    case .Branches(_, _):
```

Comment:
In cases like this, where there's nothing in the parens, you can use `case .Branches` without the parens at all.